app/bundles/CampaignBundle/Resources/views/Campaign/details.html.twig

```diff
@@ -40,6 +40,17 @@
 
 {% block publishStatus %}
   {{ include('@MauticCore/Helper/publishstatus_badge.html.twig', {'entity': campaign}) }}
+  
+  {% if campaign.allowRestart is defined and campaign.allowRestart == 1 %}
```

Comment:
```suggestion
  {% if campaign.allowRestart is defined and campaign.allowRestart is same as(1) %}
```
This ensures that the variable is defined and performs a strict type check (`same as(1)`).
Use this when we want to avoid unexpected truthy values, such as '1' or 'true'.

For general use where strict typing isn't critical, a loose comparison (== 1) is also acceptable.